include/common/nas_structs.h

```diff
@@ -50,8 +50,8 @@ struct esm_sec_info {
 #define MAX_PCO_OPTION_SIZE 260
 struct pco 
 {
-    unsigned char pco_length;
-    unsigned char pco_options[MAX_PCO_OPTION_SIZE];
+    uint16_t pco_length;
```

Comment:
pco length is really only 255. Logically we should be ok with just uint8_t

Reply:
Changed.